src/main/java/edu/kis/powp/jobs2d/drivers/SetPositionCommand.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class SetPositionCommand implements DriverCommand {
+
+    private final int x;
+    private final int y;
+    private final Job2dDriver driver;
+
+    public SetPositionCommand(Job2dDriver driver, int x, int y) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).